features/utils.py

```diff
@@ -51,7 +51,8 @@ def init_remote_firefox_browser():
     profile.set_preference('network.negotiate-auth.trusted-uris', 'https://')
     op = Options()
     op.profile = profile
-    op.add_argument("-headless")
+    if 'CI' in os.environ:
```

Comment:
also you need to add the `import os` at the top

Reply:
os module is already imported, https://github.com/RedHatProductSecurity/osim/pull/375/files#diff-e1542493f59bf0599bc9603c252d7d65e2bb13a000918b15f5a9adbfee4feb97L2

Reply:
Oh you're right, I didn't saw it